scripts/kconfig/hardenconfig.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2019 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+
+from kconfiglib import Symbol, standard_kconfig, standard_config_filename
+
+import csv
+import os
+
+
+def hardenconfig(kconf):
+    kconf.load_config()
+
+    hardened_kconf_filename = os.environ['ZEPHYR_BASE'] + os.sep +  \
+        "scripts" + os.sep + "kconfig" + os.sep + "hardened.csv"
+
+    options = compare_with_hardened_conf(kconf, hardened_kconf_filename, {})
+
+    display_results(options)
+
+
+class Option(object):
+
+    def __init__(self, name: str, recommended: str, current=None, symbol=None):
+        self.name = name
+        self.recommended = recommended
+        self.current = current
+        self.symbol = symbol
+
+        if current is 'NA':
```

Comment:
Safer to use `==` when comparing strings. `is` compares object identity (but will often work for strings due to Python internals).

Reply:
fair